zilliqa/src/cfg.rs

```diff
@@ -260,10 +269,17 @@ pub fn scilla_address_default() -> String {
     String::from("http://localhost:3000")
 }
 
-pub fn scilla_lib_dir_default() -> String {
+pub fn scilla_stdlib_dir_default() -> String {
     String::from("/scilla/0/_build/default/src/stdlib/")
 }
 
+pub fn scilla_ext_libs_cache_folder_default() -> ScillaExtLibsCacheFolder {
+    ScillaExtLibsCacheFolder {
+        on_host: String::from("/tmp"),
+        on_docker: String::from("/scilla_ext_libs"),
```

Comment:
Because the containerised version of Scilla needs to know where to look for libraries and its filesystem looks different to the host's.

(and z2 does indeed run Scilla natively - once this PR is merged, I'll add a hack - because obviously every node needs a different directory or they will conflict).

Reply:
I believe the best approach is the one suggested by @rrw-zilliqa: adopting a stateless design. This involves sending the external libraries to the Scilla HTTP server along with the contract’s code. However, for now, we're not going to touch the Scilla server so we need to inform the Scilla server where to locate the external libraries. We can’t use the same configuration for both the dockerized version of the Scilla server and the one executed locally on the zq2 server.